dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/SSLTest.java

```diff
@@ -129,9 +130,26 @@ private void configureBeforeCheckpoint() throws Exception {
             case testTransportSecurityConfigured:
                 addKeystore("serverKeyStore", "server-keystore.jks", "JKS", "secret");
                 addKeystore("serverTrustStore", "server-truststore.jks", "JKS", "secret");
-                addSSL("defaultSSLConfig", "serverKeyStore", "serverTrustStore");
-                LibertyServerFactory.recursivelyCopyDirectory(server.getMachine(), new LocalFile("lib/LibertyFATTestFiles/sslKeystore/resources"),
-                                                              server.getMachine().getFile(server.getServerRoot() + "/resources"));
+                configureSSL();
+                break;
+            case testSSLConfiguredEncryptedAES128:
+            case testTransportSecurityConfiguredEncryptedAES128:
+                addKeystore("serverKeyStore", "server-keystore.jks", "JKS",
+                            "{aes}APbuDThpBP7uIDSWGxsRc18zh+4ZNFWnMDA32eSfCvx6");
+                addKeystore("serverTrustStore", "server-truststore.jks", "JKS",
+                            "{aes}APbuDThpBP7uIDSWGxsRc18zh+4ZNFWnMDA32eSfCvx6");
+                configureSSL();
+                break;
+            case testSSLConfiguredEncryptedAES256:
+            case testTransportSecurityConfiguredEncryptedAES256:
+                addKeystore("serverKeyStore", "server-keystore.jks", "JKS",
+                            "{aes}ARDGSfGkWA6n8Y+LdjyeNvBBxnNAx0nayjV/gcPC7NSXtekgpU/5ILdXQ+Kx6NaX6UTyBBoOL/U+1nSHy9JiVptQP394y39iiWd6ctVEFtBOMOYuojK5O8osEBrWLG+ECDFKvCQRcRo5");
```

Comment:
```suggestion
                            "{aes}ARATaq8k5SQc6VBaw+IB9LmGg5R3ZvpsriWJTKOF7ty3F6mKcfCasPndBRD07yGyrJ7hJaCRD/NiibtkzqcPOxwudqsqLV3/q1Ucjjfh4XpeaEw3xOhY99JGlMxtFoPDDa7i/YHva5bU"); //secret
```
Because we just merged #30682. The AES-256 passwords need to be regenerated. I have regenerated yours above.

Reply:
While we are making these changes, can we put in comments what the plain text password is that got encrypted here and also can you put in comments the command used to do that encryption.  That way someone fresh to this config can easily understand what this generated value is and where it came from?

Reply:
agreed 👍 

wlp\bin>securityUtility encode --encoding=aes secret
{aes}ARATaq8k5SQc6VBaw+IB9LmGg5R3ZvpsriWJTKOF7ty3F6mKcfCasPndBRD07yGyrJ7hJaCRD/NiibtkzqcPOxwudqsqLV3/q1Ucjjfh4XpeaEw3xOhY99JGlMxtFoPDDa7i/YHva5bU

Reply:
also a reminder, each time you run the command you'll get a different encrypted string output, that is by design and is expected.